_includes/posts/listing/standard.html

```diff
@@ -30,6 +30,11 @@ <h1 itemprop="headline" class="blog-posting__headline">{{ include.post.title }}<
             </div>
             {% endif %}
         </div>
+        {% if include.post.youtube_src %}
+        <div class="blog-posting__media" >
+            {% include youtube.html id='uIBNFVG2gi4' width='100%' %}
```

Comment:
Why not use `youtube_id` instead of `youtube_src` and just use the identifier accordingly? This seems hardcoded.

Reply:
yes i missed that point, Thanks.